extensions/componentdsl/runtime/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.camel.quarkus</groupId>
+        <artifactId>camel-quarkus-componentdsl-parent</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>camel-quarkus-componentdsl</artifactId>
+    <name>Camel Quarkus :: Component DSL :: Runtime</name>
+    <description>Create Camel components with a fluent Java DSL</description>
+
+    <properties>
+        <firstVersion>1.0.0-M3</firstVersion>
```

Comment:
```suggestion
        <firstVersion>1.0.0-CR3</firstVersion>
```

Reply:
I will update this but wonder if it does make any sense to set the first version to a pre 1.0 release, shouldn’t we update all the first version to 1.0 once we release it ?

Reply:
Yes they should all be 1.0.0 when its GA. All older versions are EOL and dead. And those M2, RC2, RC3 versions dont add value but just more confusion.

Reply:
It is still practical to be able to tell (for a tweet and/or blog post) what's new in 1.0.0 compared to CR2 for the users who know/use CR2.

Reply:
So I'd vote for consistently using the CR3 label for the newest stuff until shortly before the release and eliminate all labels when releasing.

Reply:
I think no one is questioning about having CR3 here now but the question was about what we should do when we'll release 1.0

Reply:
Yes, changing all firstVersions to 1.0.0 is a good idea. Sorry that I have not expressed that straight away.